pandas/tools/plotting.py

```diff
@@ -2262,11 +2262,13 @@ def result(self):
 
 def _plot(data, x=None, y=None, subplots=False,
           ax=None, kind='line', **kwds):
+
+    invalid_ct_fmtstr = "'%s' is not a valid chart type"
```

Comment:
Hmm... I don't really have a strong opinion here about the exact wording.

In any case this is a clear improvement over the original. If you're happy with the wording that's fine.